smalltalksrc/Slang/TMethod.class.st

```diff
@@ -1678,6 +1679,7 @@ TMethod >> inlineSend: aSendNode directReturn: directReturn exitVar: exitVar in:
 				 in: aCodeGen);
 		addAll: meth statements. "method body"
 	directReturn ifTrue: [
+		meth parseTree parent: aSendNode parent.
```

Comment:
🤔  Is this ok? The parent of a send is not the method? So the parent of `meth` is another method?

Reply:
Let's open an issue for this